app/views/idv/otp_verification/show.html.slim

```diff
@@ -0,0 +1,31 @@
+- title t('titles.enter_2fa_code')
+
+h1.h3.my0 = t('devise.two_factor_authentication.header_text')
+
+p == @presenter.phone_number_message
+
+= form_tag(:idv_otp_verification, method: :put, role: 'form', class: 'mt3') do
+  = label_tag 'code', \
+    t('simple_form.required.html') + t('forms.two_factor.code'), \
+    class: 'block bold'
+  .col-12.sm-col-5.mb2.sm-mb0.sm-mr-20p.inline-block
+    = text_field_tag(:code, '', value: @code, required: true,
+      autofocus: true, pattern: '[0-9]*', class: 'col-12 field monospace mfa',
+      'aria-describedby': 'code-instructs', maxlength: Devise.direct_otp_length,
+      autocomplete: 'off', type: 'tel')
+  = submit_tag t('forms.buttons.submit.default'), class: 'btn btn-primary align-top sm-col-6 col-12'
+  br
+  br
+= button_to(t('links.two_factor_authentication.get_another_code'), idv_resend_otp_path,
+    method: :post,
+    class: 'btn btn-link btn-border ico ico-refresh text-decoration-none',
+    form_class: 'inline-block')
+
+br
+
+p = link_to t('forms.two_factor.try_again'), idv_phone_path
+p = link_to t('two_factor_authentication.choose_another_option'), idv_otp_delivery_method_url
+
+.mt3.border-top
+  .mt1
+    = link_to t('links.cancel'), idv_cancel_path
```

Comment:
It looks like this has changed the UI a bit compared to what's on master. Here is what's on master:
<img width="695" alt="screen shot 2018-08-31 at 2 50 36 pm" src="https://user-images.githubusercontent.com/811150/44930854-9f25fa80-ad2d-11e8-8d73-50c47a85c2e2.png">

And here is what's on this PR:
<img width="673" alt="screen shot 2018-08-31 at 2 33 14 pm" src="https://user-images.githubusercontent.com/811150/44930861-a947f900-ad2d-11e8-9115-62c3c9372f57.png">

Perhaps we should ask @donjo and @rtwell for design feedback?

Should we keep the "entered the wrong number" text and link and add the new "choose another option" under that?